lib/src/layer/polygon_layer/polygon.dart

```diff
@@ -49,11 +49,26 @@ class Polygon {
   /// The [TextStyle] of the [Polygon.label].
   final TextStyle labelStyle;
 
-  /// The placement logic of the [Polygon.label].
+  /// The placement logic of the [Polygon.label]
+  ///
+  /// [PolygonLabelPlacement.polylabel] can be expensive for some polygons. If
+  /// there is a large lag spike, try using [PolygonLabelPlacement.centroid].
   final PolygonLabelPlacement labelPlacement;
 
+  /// Whether to rotate the label counter to the camera's rotation, to ensure
+  /// it remains upright
   final bool rotateLabel;
 
+  /// {@macro fm.PolygonLayer.performantRendering}
+  ///
+  /// Value meanings (defaults to `true`):
+  ///
+  /// - `true` : respect layer-level value (disabled by default)
+  /// - `false`: disabled, ignore layer-level value
+  ///
+  /// Also see [PolygonLayer.performantRendering].
+  final bool performantRendering;
```

Comment:
Do we actually have users that want to switch the rendering logic per polygon but have them together in one PolygonLayer? I'm just worried that this introduces unnecessary complexity without common or rare usage.

If so there should be a hint in the API docs to `PolygonLayer.performantRendering` (not just in the online documentation).

Reply:
I've removed the feature level argument altogether. Resolved in 9475c7a.